ipld/kamt/fuzz/fuzz_targets/common.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019-2022 ChainSafe Systems
```

Comment:
???

Reply:
Copy of https://github.com/filecoin-project/ref-fvm/blob/master/ipld/hamt/fuzz/fuzz_targets/common.rs

I created that file, but only as a refactoring of code that was already in `simple.rs` :man_shrugging: